src/components/DaysButtons/DaysButtons.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+
+const DaysButtons = () => {
+  const [activeButton, setActiveButton] = useState('');
+
+  const handleClick = (id) => {
+    setActiveButton(id);
+  };
+
+  return (
+    <div className='div-for-days'>
+      <div className='day-menu'>
+        <button
```

Comment:
You can put this in separate component so it is called DayButton for example. You can see here that they are very similar.